azure-ts-static-website/README.md

```diff
@@ -0,0 +1,56 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# A Static Website Hosted on Azure Blob Storage + Azure CDN
+
+This example configures [Static website hosting in Azure Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website). One complication is the fact that the Static Website feature of Storage Accounts is not part of Azure Resource Manager, and is not configurable directly via Pulumi Azure provider.
+
+As a workaround, a custom dynamic provider and a dynamic resource are created. The provider delegates the setup to Azure CLI commands, while still providing Pulumi experience and lifecycle management.
```

Comment:
Perhaps link "dynamic provider" to https://pulumi.io/reference/programming-model/#dynamicproviders?